sdk/core/Azure.Core.TestFramework/src/Azure.Core.TestFramework.csproj

```diff
@@ -1,6 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFrameworks>$(RequiredTargetFrameworks);net47</TargetFrameworks>
+    <NoWarn>$(NoWarn);SYSLIB0023</NoWarn>
```

Comment:
I think we should add these to our own project files, not to the Azure.Core.TestFramework project file.

Reply:
Than can you address the warning on a line by line basis?

Reply:
Changed to only suppress the line.